bundles/org.openhab.binding.electroluxair/src/main/java/org/openhab/binding/electroluxair/internal/handler/ElectroluxAirBridgeHandler.java

```diff
@@ -147,6 +150,15 @@ private void stopAutomaticRefresh() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        return;
+        if (CHANNEL_STATUS.equals(channelUID.getId()) || command instanceof RefreshType) {
+            ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
```

Comment:
Why do you need to create a new thread pool for this?

Reply:
Fixed!